src/proto/project.proto

```diff
@@ -75,6 +75,16 @@ message DeleteProjectRequest {
   optional string owner_id = 2; // The ID of the organization owning the project, empty for default org for user
 }
 
+message QueryAssetsRequest {
+  string id = 1;                     // ID of the project to request, UUIDv4, empty string uses default project
+	optional string owner_id = 2;      // The ID of the organization owning the assets, empty for default org for user
+	optional uint64 since = 3;         // The unix time to start listing assets from, empty for no restriction
+	optional uint64 until = 4;         // The unix time to stop listing assets at, empty for no restriction
+	optional uint64 limit = 5;         // The maximum number of assets to return, empty for no limit
+	optional uint64 offest = 6;        // The starting offset when returning assets, empty for no offset
```

Comment:
Cursor based pagination would be more efficient in both postgres and dynamodb (if we do decide to go that route later)

Reply:
This should be addressed by the latest change.